doc/release/release_0.17.rst

```diff
@@ -0,0 +1,371 @@
+  example notebook on a binder instance. (#4543)
+
+New doc tutorials and gallery examples have been added to the use of regionprops_table in (#4348)
+geometrical transformations (#4385), and the registration of rotation and
+scaling with no shared center (#4515). A new section on registration has been
+added to the gallery (#4575).
+
+Improvements
+------------
+
+- scikit-image aims at being fully compatible with 3D arrays, and when possible
+  with nD arrays. nD support has been added to color conversion functions
+  (#4418), to the CLAHE `exposure.equalize_adapthist` algorithm (#4598) 
+  and to the Sobel, Scharr, and Prewitt filters (#4347).
+- Multichannel support for denoise_tv_bregman (#4446)
+- The memory footprint of `segmentation.relabel_sequential` has been reduced in
+  the case of labels much larger than the number of labels (#4612)
+- Random ellipses are now possible in `draw.random_shapes` (#4493)
+- Add border conditions to ridge filters (#4396)
+- `segmentation.random_walker` new Jacobi preconditioned conjugate gradient mode
+  (#4359) and minor corrections #4630
+- Masked marching cubes (#3829)
+- Update rescale_intensity to prevent under/overflow and produce proper output dtype (#4585)
+- Warn when rescaling with NaN in exposure.intensity_range (#4265)
+
+We have also improved the consistency of several functions regarding the way
+they handle data types
+
+- Make dtype consistent in filters.rank functions (#4289)
+- Fix colorconv float32 to double cast (#4296)
+- Prevent radon from upcasting float32 arrays to double (#4297)
+- Manage iradon_sart input and output data type (#4300)
+
+API Changes
+-----------
+- When used with floating point inputs, ``denoise_wavelet`` no longer rescales
+  the range of the data or clips the output to the range [0, 1] or [-1, 1].
+  For non-float inputs, rescaling and clipping still occurs as in prior
+  releases (although with a bugfix related to the scaling of ``sigma``).
+- For 2D input, edge filters (Sobel, Scharr, Prewitt, Roberts, and Farid)
+  no longer set the boundary pixels to 0 when a mask is not supplied. This was
+  changed because the boundary mode for `scipy.ndimage.convolve` is now
+  ``'reflect'``, which allows meaningful values at the borders for these
+  filters. To retain the old behavior, pass
+  ``mask=np.ones(image.shape, dtype=bool)`` (#4347)
+- When ``out_range`` is a range of numbers and not a dtype in
+  :func:`skimage.exposure.rescale_intensity`, the output data type will always
+  be float (#4585)
+- The values returned by :func:`skimage.exposure.equalize_adapthist` will be
+  slightly different from previous versions due to different rounding behavior
+  (#4585)
+- Move masked_register_translation from feature to registration (#4503)
+- Move register_translation from skimage.feature to skimage.registration (#4502)
+- Move watershed from morphology to segmentation (#4443)
+- Rename draw.circle() to draw.disk() (#4428)
```

Comment:
We need to mention #4612 here. I thought I had added this line, I guess I forgot! Oops!
```suggestion
- Rename draw.circle() to draw.disk() (#4428)
- The forward and backward maps returned by :func:`skimage.segmentation.relabel_sequential`
  are no longer NumPy arrays, but more memory-efficient `ArrayMap` objects that behave
  the same way for mapping. See the ``relabel_sequential`` documentation for more details.
  To get NumPy arrays back, cast it as a NumPy array: ``np.asarray(forward_map)`` (#4612)
```